### ISSUE TITLE
hide comment //Input var Okay.

### ISSUE THREAD

**Issue by hiddenpearls** [WordPress-Coding-Standards/WordPress-Coding-Standards#511]
For all the $_POST warnings. I am using the comment

```
//Input var Okay.
```

Is there any way that I don't write this comment on everyline but instead of some sort of settings in PHPCS or WPCS ?

**Comment by JDGrimes** [comment]
You can create a custom ruleset and exclude that rule:

```xml
<?xml version="1.0"?>
<ruleset name="Custom">
	<description>My custom ruleset</description>

	<rule ref="WordPress">
		<exclude name="WordPress.VIP.SuperGlobalInputUsage" />
	</rule>
</ruleset>
```

This is what I do, since there are several rules in the `WordPress-VIP` and `WordPress-Extra` rulesets that I don't use.

**Comment by hiddenpearls** [comment]
@JDGrimes When I go to /wpcs/WordPress/ruleset.xml  and should I replace the file content with the content you provided or ??

```
<ruleset name="Custom">
    <description>My custom ruleset</description>

    <rule ref="WordPress">
        <exclude name="WordPress.VIP.SuperGlobalInputUsage" />
    </rule>
</ruleset>
```

Whatever I do, It actually doesn't show me any error. Though I just want to hide the //Input Var Okay. comments warnings.

**Comment by GaryJones** [comment]
@hiddenpearls You'd use the content JDGrimes gave you, and put it into a ruleset.xml or phpcs.xml, then ensure PHP_CodeSniffer uses this new file.
Do not edit the code from this repo.

**Comment by hiddenpearls** [comment]
@GaryJones How I can make sure which ruleset.xml is being used by PHP_CodeSniffer ?

**Comment by GaryJones** [comment]
If it's in the root of your project, it will find it. Or you can pass it as a flag when calling it on the command line, or set it as a setting in your IDE etc.

**Comment by JDGrimes** [comment]
I've added this information to the [wiki](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/wiki/Flagged-superglobal-usage-in-WordPress-VIP).

**Comment by hiddenpearls** [comment]
@GaryJones I put it in the root of the project and didn't worked. So I used it as a flag and my PHPCS command was like this.

```
phpcs --standard=WordPress --standard=/applications/mamp/htdocs/ruleset.xml  /applications/mamp/htdocs/wptest2/wp-content/plugins/wp-analytify/inc/analytify-settings.php
```

I used 2 flags. Is it Ok ?

@JDGrimes Thanks for adding this information. It is really useful but may be add the full command like above.

Thanks!
